OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -528,6 +548,41 @@ public bool CanRepairAt(Actor host)
 			return repairable != null && repairable.Info.RepairActors.Contains(host.Info.Name) && self.GetDamageState() != DamageState.Undamaged;
 		}
 
+		bool IsBlockedBy(Actor self, Actor otherActor, Actor ignoreActor)
+		{
+			// We are not blocked by the actor we are ignoring.
+			if (otherActor == ignoreActor)
+				return false;
+
+			// If self is null, we don't have a real actor - we're just checking what would happen theoretically.
+			// In such a scenario - we'll just assume any other actor in the cell will block us by default.
+			// If we have a real actor, we can then perform the extra checks that allow us to avoid being blocked.
+			if (self == null)
```

Comment:
This is inside `Aircraft` (not `AircraftInfo`), so `self` is guaranteed by convention to be non-null.

Reply:
Oops. That's what I get for shamelessly copy-pasting. Fixed.